lib/src/Pos/Launcher/Resource.hs

```diff
@@ -319,7 +319,7 @@ createKademliaInstance ::
     -> Word16 -- ^ Default port to bind to.
     -> m KademliaDHTInstance
 createKademliaInstance BaseParams {..} kp defaultPort =
-    usingLoggerName (lpRunnerTag bpLoggingParams) (startDHTInstance instConfig defaultBindAddress)
+    usingLoggerName (lpDefaultName bpLoggingParams) (startDHTInstance instConfig defaultBindAddress)
```

Comment:
I think that it'd better use a dedicated `kademlia` logging name instead.

Reply:
I preserved existing logic, but ok, I think it makes sense indeed.